docs/api/golang.mdx

```diff
@@ -0,0 +1,303 @@
+  "timestamp_raw": 1737743855312976962
+}
+```
+
+As expected, only events from `mycontainer` are captured and the runtime node
+contains all information related to the container creating the events. Also, a
+new timestamp field with a human friendly version is present.
+
+Don't forget to remove the containers created above:
+
+```bash
+$ docker rm mycontainer foocontainer
+```
+
+### Data Source
+
+So far, we've been using the json formatter to convert the events produced by a
+Gadget to json. However, if you're only interested on getting specific fields,
+it can be more performant to access them directly using a Field Accessor instead
+of performing a full json marshaling and unmarshaling.
+
+import Datasource from '!!raw-loader!./_gadgets/datasource/main.go';
+
+<CodeBlock language="go">{Datasource}</CodeBlock>
+
+Executing this program will output the following:
+
+```bash
+$ go run -exec sudo .
+command systemd-journal (761) opened /run/log/journal/02962d410aef4be9b1be7d022e8be679/system.journal
+command systemd-journal (761) opened /run/log/journal/02962d410aef4be9b1be7d022e8be679/system.journal
+command tmux: server (12404) opened /proc/38122/cmdline
+command systemd-oomd (1245) opened /proc/meminfo
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.pressure
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.current
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.min
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.low
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.swap.current
+command systemd-oomd (1245) opened /sys/fs/cgroup/user.slice/user-1001.slice/user@1001.service/memory.stat
+command systemd-oomd (1245) opened /proc/meminfo
+command dbus-daemon (3811) opened /sys/kernel/security/apparmor/.access
+command tmux: server (12404) opened /proc/38122/cmdline
+command systemd-oomd (1245) opened /proc/meminfo
+command systemd-oomd (1245) opened /proc/meminfo
+command tmux: server (12404) opened /proc/38122/cmdline
+```
+
+### Running Gadgets remotely with the gRPC runtime
+
+All the examples above run the eBPF locally. This example shows how to control a
+remote `ig` instance by using the gRPC runtime. In this case, the Gadget image
+is pulled by the remote instance, hence the client doesn't need to contact the
+OCI registry directly.
+
+import GRPC from '!!raw-loader!./_gadgets/grpc/main.go';
+
+<CodeBlock language="go">{GRPC}</CodeBlock>
+
+In this case we need to run `ig` on daemon mode:
+
+```bash
+$ sudo ig daemon --host tcp://127.0.0.1:8888
+INFO[0000] starting Inspektor Gadget daemon at "tcp://127.0.0.1:8888"
+```
+
+Now run the example. Note that in this case root is not needed because the all
+privileged operations are done by the `ig` process.
```

Comment:
```suggestion
Now run the example. Note that in this case running as user "root" is not needed, because all
the privileged operations are happening inside the `ig` process on the remote side.
```